tests/fast-integration/eventing-test/utils.js

```diff
@@ -143,6 +144,30 @@ async function getJetStreamStreamData(host) {
   };
 }
 
+async function getSubscriptionConsumerName(subscriptionName, namespace='default', crdVersion='v1alpha1') {
+  const sub = await getSubscription(subscriptionName, namespace, crdVersion);
```

Comment:
We can move it to the `else` block, as it is not used in the if block, so we can avoid an unnecessary api call.